apps/full-site-editing/package.json

```diff
@@ -82,10 +82,12 @@
 		"prepare": "check-npm-client && yarn run sync:newspack-blocks --nodemodules"
 	},
 	"dependencies": {
-		"@automattic/format-currency": "^1.0.0-alpha.0",
-		"@automattic/data-stores": "^1.0.0-alpha.1",
 		"@automattic/calypso-build": "*",
+		"@automattic/data-stores": "^1.0.0-alpha.1",
+		"@automattic/domain-picker": "1.0.0-alpha.0",
+		"@automattic/format-currency": "^1.0.0-alpha.0",
 		"@wordpress/api-fetch": "*",
+		"@wordpress/base-styles": "1.9.0",
```

Comment:
Likely needs `^`

Reply:
PRed https://github.com/Automattic/wp-calypso/pull/43058 since there seems to be NPM config problem.